### ISSUE TITLE
Inference: detect serializer for associations

### ISSUE THREAD

**Issue by yosiat** [yosiat/panko_serializer#7]
I want to be able not to specify specific serializers in associations and it will find automatically.

```
class PostSerailizer < Panko::Serializer
  has_many :comments
end
```

it will use `CommentsSerializer` here

**Comment by yosiat** [comment]
will be released in the next version.
@oyeanuj FYI

**Comment by yosiat** [comment]
@oyeanuj released in 0.3.3 - https://github.com/yosiat/panko_serializer/releases/tag/v0.3.3
